conda_smithy/feedstock_content/ci_support/upload_or_check_non_existence.py

```diff
@@ -57,17 +58,23 @@ def upload(cli, meta, owner, channels):
         os.remove('binstar.token')
 
 
-def distribution_exists_on_channel(binstar_cli, meta, owner, channel='main'):
+def distribution_exists_on_channel(binstar_cli, owner, metadata, channel='main'):
```

Comment:
This is breaking the API elsewhere and doesn't match what we are doing in other places in this code. So am going to adjust the code to match.